manager/cmd/version.go

```diff
@@ -0,0 +1,90 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
```

Comment:
version.go is usually used to hold the version of the program.  gke-version??

Also headers

Reply:
Fixed.